paraview/src/CloudberryDemo.js

```diff
@@ -0,0 +1,437 @@
+/* global document */
```

Comment:
paraview/src/CloudberryDemo.js is the source code and paraview/dist/CloudberryDemo.js is the code that is generated by webpack, which combines the source code with all of the required libraries into one JavaScript file. The *.woff2 and *.ttf are generated by webpack, too. I leave these files there because I think it would be easier for people who just want to try it but does not want to modify the source code. I can delete them if they are not necessary.